src/app/api/uploadgar/validations.js

```diff
@@ -5,6 +5,14 @@ const airportValidation = require('../../../common/utils/airportValidation');
 const validator = require('../../../common/utils/validator');
 const ValidationRule = require('../../../common/models/ValidationRule.class');
 const { MAX_STRING_LENGTH, MAX_REGISTRATION_LENGTH } = require('../../../common/config/index');
+const documenttype = require('../../../common/seeddata/egar_saved_people_travel_document_type.json');
+
+
+const documentTypes = documenttype
+    .map(documentType => documentType.documenttype)
+    .filter(documentType => Boolean(documentType))
```

Comment:
These lines seems to be a repeat of the ones in utils/validator.js, can they be refactored?

Reply:
This has been done.